enhancements/cluster-logging/hypershift-audit-logs.md

```diff
@@ -21,174 +21,124 @@ see-also:
+ - Receiver should be identified with a special input name e.g. `hypershiftAPIAudit`
+ - Pipelines from the receiver may include an [api-audit-log-policy](/api-audit-log-policy.md) filter.
+1. Forward logs to any outputs as desired.
 
 #### Management Cluster, control plane
 
-On the _management_ cluster
-1. Install the Cluster Logging Operator (CLO), watching all name-spaces.
-1. Install the new Hypershift Logging Operator (HLO), watching all name-spaces.
-1. In each new HCP name-space, create a `ClusterLogForwarderTemplate` resource. \
-   This resource may be identical for all HCPs or customized per HCP.
+1. Install the Cluster Logging Operator (CLO).
+1. Create a `ClusterLogForwarder` for each HCP to forward to desired outputs.
+1. Install the new Hypershift Logging Operator (HLO) watching each HCP namespace.
+   - The HLO watches CLF configurations _in the data plane_, not the control plane.
+   - When the HLO detects a _data plane_ CLF requesting audit log forwarding, it updates
+     the _control plane_ CLF for that HCP to forward audit logs via a HTTP output to the _data plane_ CLF.
+   - The HLO should copy audit log policies from  _data plane_ CLF pipelines to the _control plane_,
+     to forward only those logs that will also be forwarded by the data plane CLF.
 
 See "API Extensions" and "Implementation Details" for more.
 
 ### API Extensions
 
-#### HypershiftLogForwarder
-
-The `HyperShiftLogForwarder` resource is created on the data plane (_hosted_ cluster),
-but is reconciled by the HLO on the _management_ cluster.
-
-This allows both service consumer and provider to create forwarding configuration,
-without exposing service provider configuration to the consumer.
-
-`HyperShiftLogForwarder` (HFL) is identical to the `ClusterLogForwarder` (CLO )API except that:
-- Only the `audit` input is allowed, and only API server audit logs will be forwarded.
-- Any pipeline or output features that cannot be used with an `audit` input are disabled.
-- The HLF resource is created on the _hosted_ cluster, but describes forwarding for the HCP on the _management_ cluster.
-
-#### ClusterLogForwarderTemplate
-
-The `ClusterLogForwarderTemplate` is created in a HCP name-space on the _management_ cluster.
-It contains a `ClusterLogForwarder` for the service provider.
-
-Reconciled by the _Hypershift Logging Operator_ on the _management_ cluster.
-- Watch `HyperShiftLogForwarder.spec` in the _hosted_ cluster:
-  - Prefix input, output and pipeline names from the `HyperShiftLogForwarder` with "hosted_".
-  - Append "hosted_" items to the `ClusterLogForwarderTemplate`
-  - Create a new `ClusterLogForwarder` in the HCP containing the combined data.
-- Watch `ClusterLogForwarder.status` in the HCP name-space
-  - Propagate error conditions to the `HyperShiftLogForwarder.status` in the _hosted_ cluster (remove "hosted_" prefix).
+None. The HLO watches existing resources: HCP on the control-plane, CLF on the data plane.
```

Comment:
If there is no new custom resource defined, is it necessary to be an operator here?

Reply:
How will a workload be deployed in either cluster without something reacting to intent by the admin? Operators are designed to manage a workload based upon a spec. The alternative is the admin crafts every needed resource and deploys it and reverts or modifies it when changes happen to them, intentional or not

Reply:
@jcantrill this is an unusual case. The HLO operator does not have any resources of its own. It  observes CLF resources  _in another cluster_ which is possible because the data-plane API server actually runs on the management cluster.  It modifies CLF resources in its own cluster to create the HTTP connection from management-CLF to guest-CLF.  It does not actually implement any of the forwarding behavior itself, it just observes and congifgures CLF resources and lets the CLO do it's normal job of enacting the configuration.